src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -32,8 +32,7 @@ info:
 
     There is no API versioning, since there is no need for it right now.
 
-    Only rudimentary rate limiting is implemented, so please be gentle when using the API concurrently, especially with potentially expensive operations.
-    In case of abuse, we will limit/remove your access.
```

Comment:
Hm I would not remove this. Just add the sentence below.

Reply:
This text is documentation that will ship with each instance of the code. As the commit says "Specifics of rate limiting or other authentication schemes should not be detailed in the general API documentation.".

Reply:
This line "Only rudimentary rate limiting is implemented, so please be gentle when using the API concurrently, especially with potentially expensive operations. In case of abuse, we will limit/remove your access." should be moved to some place dependant on the OBS instance. For example, in the `Description` field which can be modified under `/configuration`:

![Screenshot from 2022-07-29 13-12-41](https://user-images.githubusercontent.com/24919/181747441-14e4fcea-54ce-4f59-b0c9-27647659d752.png)

Reply:
I think it's right to remove that line. There is nothing at application level that matches that statement, so technically that is not true...